packages/legend-studio/src/stores/shared/__tests__/MockDataUtil.test.ts

```diff
@@ -113,3 +117,15 @@ test(unitTest('Test mock data with classes cycle'), () => {
   // should not continue on to next depth
   expect(secondApplicantInstance).not.toContain('previousApplication');
 });
+
+test(unitTest('Generate default value for primitive parameter'), () => {
```

Comment:
no! Let's create a separate test for these in `legend-graph`, something like `src/helpders/__tests__/DomainHelper.test.ts`